runtime/vm/JVMImage.cpp

```diff
@@ -22,10 +22,14 @@
 
 #include "JVMImage.hpp"
 
-#define OMRPORT_FROM_IMAGE JVMImage::getJVMImage()->getPortLibrary();
+#define OMRPORT_FROM_IMAGE JVMImage::getInstance()->getPortLibrary();
+
+JVMImage *JVMImage::_jvmInstance = NULL;
+const char *JVMImage::_dumpFileName = "jvm_image.data";
```

Comment:
if you may it `static const` the compiler should treat it as a compile time constant